vignettes/04-synthesis-data.Rmd

```diff
@@ -9,8 +9,8 @@ The second analysis compares greenness from image data with canopy cover.
 ## Get and join data
 
 Here we combine two dataframes. 
-The first contains all the canopy height values for 2017, which was created in the traits vignette. 
-The second is the cumulative growing degree days for all of 2017, which were calculated from the daily minimum and maximum temperatures in the weather vignette. 
+The first contains all the canopy cover values for 2018, which was created in the traits vignette. 
```

Comment:
@dlebauer weather and canopy cover data for 2018 don't overlap. Weather data is only currently available for January - April 2018 while canopy cover data is for May - July 2018.

Reply:
And canopy cover data is only in May for 2017, though there is full weather data for that year.

Reply:
Can you ask @tcnichol to check on the 2018 weather data? It should be available in geostreams.

Reply:
2018 weather data in now available.